[0m12:12:12.204770 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffade7ab50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffae49fa10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffade7bdd0>]}


============================== 12:12:12.206404 | 092a9d87-14c2-4285-aeea-ef1a45e7ff48 ==============================
[0m12:12:12.206404 [info ] [MainThread]: Running with dbt=1.7.4
[0m12:12:12.207005 [debug] [MainThread]: running dbt with arguments {'write_json': 'True', 'introspect': 'True', 'fail_fast': 'False', 'target_path': 'None', 'debug': 'False', 'use_experimental_parser': 'False', 'send_anonymous_usage_stats': 'True', 'invocation_command': 'dbt init', 'static_parser': 'True', 'partial_parse': 'True', 'log_cache_events': 'False', 'profiles_dir': '/home/jovyan/.dbt', 'cache_selected_only': 'False', 'quiet': 'False', 'no_print': 'None', 'log_path': 'logs', 'printer_width': '80', 'use_colors': 'True', 'version_check': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'warn_error': 'None', 'indirect_selection': 'eager', 'log_format': 'default'}
[0m12:12:12.207567 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': PosixPath('/home/jovyan/.dbt')}
[0m12:12:12.208009 [info ] [MainThread]: Creating dbt configuration folder at 
[0m12:12:17.635350 [debug] [MainThread]: Starter project path: /opt/conda/lib/python3.11/site-packages/dbt/include/starter_project
[0m12:12:17.679222 [info ] [MainThread]: 
Your new dbt project "omop" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m12:12:17.679922 [info ] [MainThread]: Setting up your profile.
[0m12:12:20.233559 [info ] [MainThread]: Profile omop written to /home/jovyan/.dbt/profiles.yml using target's sample configuration. Once updated, you'll be able to start developing with dbt.
[0m12:12:20.234533 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 8.017792, "process_user_time": 1.176123, "process_kernel_time": 0.105189, "process_mem_max_rss": "136760", "process_out_blocks": "8", "process_in_blocks": "0"}
[0m12:12:20.235084 [debug] [MainThread]: Command `dbt init` succeeded at 12:12:20.234988 after 8.02 seconds
[0m12:12:20.235577 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffade89690>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffae486750>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffb3d48690>]}
[0m12:12:20.236104 [debug] [MainThread]: Flushing usage events
[0m18:37:57.209450 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff88692a10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff88692210>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff88d956d0>]}


============================== 18:37:57.211023 | 56a6b7df-5592-48d7-9582-7849a66da81d ==============================
[0m18:37:57.211023 [info ] [MainThread]: Running with dbt=1.7.4
[0m18:37:57.211598 [debug] [MainThread]: running dbt with arguments {'indirect_selection': 'eager', 'cache_selected_only': 'False', 'debug': 'False', 'quiet': 'False', 'target_path': 'None', 'fail_fast': 'False', 'partial_parse': 'True', 'introspect': 'True', 'printer_width': '80', 'static_parser': 'True', 'log_path': 'logs', 'no_print': 'None', 'use_experimental_parser': 'False', 'send_anonymous_usage_stats': 'True', 'invocation_command': 'dbt run -m int_person.sql', 'write_json': 'True', 'warn_error': 'None', 'use_colors': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'version_check': 'True', 'log_cache_events': 'False', 'log_format': 'default', 'profiles_dir': '/home/jovyan/.dbt'}
[0m18:37:57.212281 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /workspaces/synthea_dw/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m18:37:57.213106 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.028062833, "process_user_time": 1.1468, "process_kernel_time": 0.121766, "process_mem_max_rss": "109528", "process_out_blocks": "8", "command_success": false, "process_in_blocks": "0"}
[0m18:37:57.213606 [debug] [MainThread]: Command `dbt run` failed at 18:37:57.213499 after 0.03 seconds
[0m18:37:57.214015 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff88b24850>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff88b27050>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff88b26650>]}
[0m18:37:57.214421 [debug] [MainThread]: Flushing usage events
[0m19:53:34.094481 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffac564250>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffacb7b890>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffade36910>]}


============================== 19:53:34.095774 | 87053ddb-3c76-4b5a-ac1f-129d3d0e87ae ==============================
[0m19:53:34.095774 [info ] [MainThread]: Running with dbt=1.7.4
[0m19:53:34.096329 [debug] [MainThread]: running dbt with arguments {'debug': 'False', 'no_print': 'None', 'invocation_command': 'dbt run -m int_patient', 'write_json': 'True', 'target_path': 'None', 'fail_fast': 'False', 'log_cache_events': 'False', 'send_anonymous_usage_stats': 'True', 'use_colors': 'True', 'version_check': 'True', 'log_path': 'logs', 'profiles_dir': '/home/jovyan/.dbt', 'static_parser': 'True', 'quiet': 'False', 'introspect': 'True', 'use_experimental_parser': 'False', 'cache_selected_only': 'False', 'indirect_selection': 'eager', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'printer_width': '80', 'partial_parse': 'True', 'log_format': 'default', 'warn_error': 'None'}
[0m19:53:34.096875 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /workspaces/synthea_dw/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m19:53:34.097542 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.02508429, "process_user_time": 1.085046, "process_kernel_time": 0.08992, "process_mem_max_rss": "109400", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m19:53:34.098014 [debug] [MainThread]: Command `dbt run` failed at 19:53:34.097900 after 0.03 seconds
[0m19:53:34.098395 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffac54ca90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffac543110>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffac542f90>]}
[0m19:53:34.098771 [debug] [MainThread]: Flushing usage events
[0m15:28:11.093710 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff8fd43ed0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff8fceb190>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff8f70bb50>]}


============================== 15:28:11.095549 | 34ce224d-466c-4e61-9e4f-953e65a498b7 ==============================
[0m15:28:11.095549 [info ] [MainThread]: Running with dbt=1.7.4
[0m15:28:11.096053 [debug] [MainThread]: running dbt with arguments {'quiet': 'False', 'log_path': 'logs', 'debug': 'False', 'partial_parse': 'True', 'log_cache_events': 'False', 'warn_error': 'None', 'printer_width': '80', 'write_json': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'use_experimental_parser': 'False', 'fail_fast': 'False', 'indirect_selection': 'eager', 'version_check': 'True', 'log_format': 'default', 'use_colors': 'True', 'static_parser': 'True', 'target_path': 'None', 'send_anonymous_usage_stats': 'True', 'cache_selected_only': 'False', 'invocation_command': 'dbt run -m int_care_site', 'no_print': 'None', 'profiles_dir': '/home/jovyan/.dbt'}
[0m15:28:11.096535 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /workspaces/synthea_dw/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m15:28:11.097551 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.02653625, "process_user_time": 1.082939, "process_kernel_time": 0.126014, "process_mem_max_rss": "108900", "process_in_blocks": "95248", "command_success": false, "process_out_blocks": "0"}
[0m15:28:11.097964 [debug] [MainThread]: Command `dbt run` failed at 15:28:11.097873 after 0.03 seconds
[0m15:28:11.098288 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff8fe02710>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff8f6bcbd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff8fceb590>]}
[0m15:28:11.098603 [debug] [MainThread]: Flushing usage events
[0m07:42:23.635178 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff87a83f50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff87931d10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff873bbb10>]}


============================== 07:42:23.636755 | be04f1a5-5a8c-47f0-972a-b69930d7aa94 ==============================
[0m07:42:23.636755 [info ] [MainThread]: Running with dbt=1.7.4
[0m07:42:23.637278 [debug] [MainThread]: running dbt with arguments {'use_experimental_parser': 'False', 'target_path': 'None', 'no_print': 'None', 'version_check': 'True', 'quiet': 'False', 'profiles_dir': '/home/jovyan/.dbt', 'invocation_command': 'dbt run -m care_site.sql', 'static_parser': 'True', 'indirect_selection': 'eager', 'debug': 'False', 'use_colors': 'True', 'log_cache_events': 'False', 'log_path': 'logs', 'fail_fast': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'cache_selected_only': 'False', 'send_anonymous_usage_stats': 'True', 'introspect': 'True', 'warn_error': 'None', 'printer_width': '80', 'write_json': 'True', 'log_format': 'default', 'partial_parse': 'True'}
[0m07:42:23.637795 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /workspaces/synthea_dw/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m07:42:23.638620 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.027648374, "process_user_time": 1.116064, "process_kernel_time": 0.134799, "process_mem_max_rss": "108944", "process_in_blocks": "95768", "command_success": false, "process_out_blocks": "0"}
[0m07:42:23.639127 [debug] [MainThread]: Command `dbt run` failed at 07:42:23.639027 after 0.03 seconds
[0m07:42:23.639587 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff8736cc50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff8799be10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff8795d5d0>]}
[0m07:42:23.639977 [debug] [MainThread]: Flushing usage events
[0m15:37:23.308561 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffa20d29d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffa1a41610>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffa1a42b10>]}


============================== 15:37:23.310225 | 288de908-96e7-4203-8e67-1ca4ec38503b ==============================
[0m15:37:23.310225 [info ] [MainThread]: Running with dbt=1.7.4
[0m15:37:23.310912 [debug] [MainThread]: running dbt with arguments {'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'log_path': 'logs', 'write_json': 'True', 'profiles_dir': '/home/jovyan/.dbt', 'debug': 'False', 'static_parser': 'True', 'printer_width': '80', 'introspect': 'True', 'indirect_selection': 'eager', 'partial_parse': 'True', 'quiet': 'False', 'cache_selected_only': 'False', 'warn_error': 'None', 'log_cache_events': 'False', 'use_experimental_parser': 'False', 'use_colors': 'True', 'send_anonymous_usage_stats': 'True', 'no_print': 'None', 'log_format': 'default', 'target_path': 'None', 'invocation_command': 'dbt run -m person.sql', 'fail_fast': 'False', 'version_check': 'True'}
[0m15:37:23.311644 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /workspaces/synthea_dw/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m15:37:23.312621 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.026873834, "process_user_time": 1.127145, "process_kernel_time": 0.135498, "process_mem_max_rss": "108748", "process_in_blocks": "95504", "command_success": false, "process_out_blocks": "0"}
[0m15:37:23.313131 [debug] [MainThread]: Command `dbt run` failed at 15:37:23.313018 after 0.03 seconds
[0m15:37:23.313581 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffa1a4ced0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffa1aa5e50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffa79386d0>]}
[0m15:37:23.313985 [debug] [MainThread]: Flushing usage events
[0m16:41:16.289711 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff8ad002d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff8e1f68d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff8b1a60d0>]}


============================== 16:41:16.291218 | 2e4d710d-4575-433b-8b2a-10f05a64a9f4 ==============================
[0m16:41:16.291218 [info ] [MainThread]: Running with dbt=1.7.4
[0m16:41:16.291785 [debug] [MainThread]: running dbt with arguments {'indirect_selection': 'eager', 'use_experimental_parser': 'False', 'debug': 'False', 'no_print': 'None', 'profiles_dir': '/home/jovyan/.dbt', 'invocation_command': 'dbt run -m observation.sql', 'version_check': 'True', 'cache_selected_only': 'False', 'write_json': 'True', 'warn_error': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'static_parser': 'True', 'printer_width': '80', 'send_anonymous_usage_stats': 'True', 'introspect': 'True', 'partial_parse': 'True', 'log_cache_events': 'False', 'log_format': 'default', 'fail_fast': 'False', 'use_colors': 'True', 'log_path': 'logs'}
[0m16:41:16.292427 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /workspaces/synthea_dw/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m16:41:16.293303 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.026747832, "process_user_time": 1.157759, "process_kernel_time": 0.142586, "process_mem_max_rss": "108916", "process_in_blocks": "74728", "command_success": false, "process_out_blocks": "0"}
[0m16:41:16.293803 [debug] [MainThread]: Command `dbt run` failed at 16:41:16.293698 after 0.03 seconds
[0m16:41:16.294197 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff8ad2f6d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff8acdc790>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff8b2f3fd0>]}
[0m16:41:16.294590 [debug] [MainThread]: Flushing usage events
[0m20:58:55.163591 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffa84ade50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffa89979d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffa8c20c10>]}


============================== 20:58:55.165162 | bac56a96-2c11-47d4-be1c-d2143065cebb ==============================
[0m20:58:55.165162 [info ] [MainThread]: Running with dbt=1.7.4
[0m20:58:55.165741 [debug] [MainThread]: running dbt with arguments {'log_format': 'default', 'partial_parse': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'warn_error': 'None', 'target_path': 'None', 'profiles_dir': '/home/jovyan/.dbt', 'static_parser': 'True', 'invocation_command': 'dbt run', 'printer_width': '80', 'log_path': 'logs', 'log_cache_events': 'False', 'indirect_selection': 'eager', 'fail_fast': 'False', 'quiet': 'False', 'no_print': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'version_check': 'True', 'write_json': 'True', 'send_anonymous_usage_stats': 'True', 'cache_selected_only': 'False', 'debug': 'False'}
[0m20:58:55.166449 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /workspaces/synthea_dw/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m20:58:55.167362 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.026955333, "process_user_time": 1.111416, "process_kernel_time": 0.118724, "process_mem_max_rss": "109280", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m20:58:55.167936 [debug] [MainThread]: Command `dbt run` failed at 20:58:55.167827 after 0.03 seconds
[0m20:58:55.168349 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffa84b9250>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffa8ab5f50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffffa8bbb690>]}
[0m20:58:55.168770 [debug] [MainThread]: Flushing usage events
[0m01:14:38.572482 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff81f8c8d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff81e2f110>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff81f02710>]}


============================== 01:14:38.573946 | 65de9cdf-d8bd-4c0d-b238-22e4f28c9b05 ==============================
[0m01:14:38.573946 [info ] [MainThread]: Running with dbt=1.7.4
[0m01:14:38.574490 [debug] [MainThread]: running dbt with arguments {'version_check': 'True', 'use_experimental_parser': 'False', 'send_anonymous_usage_stats': 'True', 'write_json': 'True', 'static_parser': 'True', 'target_path': 'None', 'use_colors': 'True', 'indirect_selection': 'eager', 'no_print': 'None', 'invocation_command': 'dbt run', 'fail_fast': 'False', 'debug': 'False', 'profiles_dir': '/home/jovyan/.dbt', 'partial_parse': 'True', 'quiet': 'False', 'printer_width': '80', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'log_path': 'logs', 'cache_selected_only': 'False', 'log_format': 'default', 'warn_error': 'None', 'log_cache_events': 'False'}
[0m01:14:38.575194 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /workspaces/synthea_dw/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m01:14:38.576141 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.026467375, "process_user_time": 1.129257, "process_kernel_time": 0.093855, "process_mem_max_rss": "109268", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m01:14:38.576673 [debug] [MainThread]: Command `dbt run` failed at 01:14:38.576551 after 0.03 seconds
[0m01:14:38.577111 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff81879050>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff87776690>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff87777290>]}
[0m01:14:38.577538 [debug] [MainThread]: Flushing usage events
[0m00:05:46.016221 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff85d5e6d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff86fdd150>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff85d5e810>]}


============================== 00:05:46.018059 | 86320f6b-851b-411d-8c7e-41fc9a5f7f78 ==============================
[0m00:05:46.018059 [info ] [MainThread]: Running with dbt=1.7.6
[0m00:05:46.018615 [debug] [MainThread]: running dbt with arguments {'warn_error': 'None', 'use_experimental_parser': 'False', 'log_format': 'default', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'quiet': 'False', 'invocation_command': 'dbt init', 'log_cache_events': 'False', 'cache_selected_only': 'False', 'indirect_selection': 'eager', 'fail_fast': 'False', 'debug': 'False', 'printer_width': '80', 'use_colors': 'True', 'profiles_dir': '/home/jovyan/.dbt', 'version_check': 'True', 'introspect': 'True', 'log_path': 'logs', 'send_anonymous_usage_stats': 'True', 'write_json': 'True', 'partial_parse': 'True', 'static_parser': 'True', 'target_path': 'None', 'no_print': 'None'}
[0m00:05:46.019069 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': PosixPath('/home/jovyan/.dbt')}
[0m00:05:46.019382 [info ] [MainThread]: Creating dbt configuration folder at 
[0m00:06:05.670324 [debug] [MainThread]: Starter project path: /opt/conda/lib/python3.11/site-packages/dbt/include/starter_project
[0m00:06:05.718376 [info ] [MainThread]: 
Your new dbt project "tuva_input" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m00:06:05.719357 [info ] [MainThread]: Setting up your profile.
[0m00:06:09.737101 [info ] [MainThread]: Profile tuva_input written to /home/jovyan/.dbt/profiles.yml using target's sample configuration. Once updated, you'll be able to start developing with dbt.
[0m00:06:09.738484 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 23.723583, "process_user_time": 1.111351, "process_kernel_time": 0.159193, "process_mem_max_rss": "133984", "process_in_blocks": "146424", "process_out_blocks": "8"}
[0m00:06:09.738955 [debug] [MainThread]: Command `dbt init` succeeded at 00:06:09.738861 after 23.72 seconds
[0m00:06:09.739311 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff85d67dd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff85d67bd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff8bbc37d0>]}
[0m00:06:09.739687 [debug] [MainThread]: Flushing usage events
[0m21:27:39.598807 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff8db9b6d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff8db9b3d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff8db9b610>]}


============================== 21:27:39.600783 | e8cb1767-73d5-4c27-92a7-226f38223a0d ==============================
[0m21:27:39.600783 [info ] [MainThread]: Running with dbt=1.7.7
[0m21:27:39.601350 [debug] [MainThread]: running dbt with arguments {'partial_parse': 'True', 'static_parser': 'True', 'warn_error': 'None', 'fail_fast': 'False', 'use_colors': 'True', 'printer_width': '80', 'indirect_selection': 'eager', 'target_path': 'None', 'no_print': 'None', 'invocation_command': 'dbt init', 'send_anonymous_usage_stats': 'True', 'introspect': 'True', 'debug': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'log_cache_events': 'False', 'cache_selected_only': 'False', 'quiet': 'False', 'profiles_dir': '/home/jovyan/.dbt', 'write_json': 'True', 'version_check': 'True', 'log_format': 'default', 'log_path': 'logs', 'use_experimental_parser': 'False'}
[0m21:27:39.601844 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': PosixPath('/home/jovyan/.dbt')}
[0m21:27:39.602198 [info ] [MainThread]: Creating dbt configuration folder at 
[0m21:27:50.001698 [debug] [MainThread]: Starter project path: /opt/conda/lib/python3.11/site-packages/dbt/include/starter_project
[0m21:27:50.053562 [info ] [MainThread]: 
Your new dbt project "tuva" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m21:27:50.054481 [info ] [MainThread]: Setting up your profile.
[0m21:27:56.928363 [info ] [MainThread]: Profile tuva written to /home/jovyan/.dbt/profiles.yml using target's sample configuration. Once updated, you'll be able to start developing with dbt.
[0m21:27:56.929826 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 17.332552, "process_user_time": 1.099307, "process_kernel_time": 0.162677, "process_mem_max_rss": "131308", "process_in_blocks": "142328", "process_out_blocks": "8"}
[0m21:27:56.930320 [debug] [MainThread]: Command `dbt init` succeeded at 21:27:56.930207 after 17.33 seconds
[0m21:27:56.930719 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff93611610>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff937738d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0xffff93773910>]}
[0m21:27:56.931150 [debug] [MainThread]: Flushing usage events
